src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -104,8 +105,10 @@ private static Person createEditedPerson(Person personToEdit, EditPersonDescript
         Company updatedCompany = editPersonDescriptor.getCompany().orElse(personToEdit.getCompany());
         Set<Tag> updatedTags = editPersonDescriptor.getTags().orElse(personToEdit.getTags());
         Set<Product> updatedProducts = editPersonDescriptor.getProducts().orElse(personToEdit.getProducts());
+        SupplierStatus supplierStatus = personToEdit.getStatus();
```

Comment:
Would it better to standardise getting the updated status from editPersonDescriptor instead?